Source/CesiumRuntime/Private/CesiumMetadataValue.cpp

```diff
@@ -128,6 +128,18 @@ int64 UCesiumMetadataValueBlueprintLibrary::GetInteger64(
       Value._value);
 }
 
+uint64 FCesiumMetadataValueAccess::GetUnsignedInteger64(
```

Comment:
Nitpick: I would move this function to the end of the file, to keep all of the existing `UCesiumMetadataValueBlueprintLibrary` functions grouped together. It just helps with organization :smile: